regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -49,7 +49,7 @@ var CommentReviewView = Backbone.View.extend({
 
   serialize: function() {
     return {
-      docNumber: this.docNumber,
+      doc_number: this.docNumber,
```

Comment:
Can we please stick with the name documentId? That's how it is referred to in regulations.gov.